downstream/assemblies/central-auth/assembly-assign-hub-admin-permissions.adoc

```diff
@@ -9,10 +9,10 @@ Hub administrative users will need to be assigned the role of _hubadmin_ in orde
 
 .Procedure
 . Navigate to the `ansible-automation-platform` realm on your SSO client.
-. Under the _Manage_ section on the side navigation bar, click *Users*.
+. Under the _Manage_ section on the side navigation bar, click btn:[Users].
 . Select a user from the list by clicking their ID.
-. Click the *Role Mappings* tab.
+. Click the btn:[Role Mappings] tab.
```

Comment:
Role Mappings is a tab selection and should be just bold with an * on either side. Unfortunately, AsciiDoctor doesn't have macros for anything other than menu and button, so any UI items that are not menus or buttons should remain simply bold.